teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, _name = 'Unknown')
+    super(age, _name = 'Unknown')
```

Comment:
- [ ] In your `super` method, ensure the passed in name is what is used if any, not "Unkown".

  ```
  super (age, _name)
  ```

Reply:
Thank you so much @TedLivist  for these corrections , You have taught me so much on ruby conventions . Thank you so so much .